src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/SuggestionDescriptionPanelFromLS.kt

```diff
@@ -33,11 +40,36 @@ import kotlin.math.max
 class SuggestionDescriptionPanelFromLS(
     snykCodeFile: SnykCodeFile,
     private val issue: ScanIssue
-) : IssueDescriptionPanelBase(title = getIssueTitle(issue), severity = issue.getSeverityAsEnum()) {
+) : IssueDescriptionPanelBase(
+    title = getIssueTitle(issue),
+    severity = issue.getSeverityAsEnum()
+) {
     val project = snykCodeFile.project
+    private val unexpectedErrorMessage = "Snyk encountered an issue while rendering the vulnerability description. Please try again, or contact support if the problem persists. We apologize for any inconvenience caused.";
 
     init {
-        createUI()
+        if (pluginSettings().isGlobalIgnoresFeatureEnabled && issue.additionalData.details != null) {
+            if (!JBCefApp.isSupported()) {
+                val statePanel = StatePanel(SnykToolWindowPanel.SELECT_ISSUE_TEXT)
+                this.add(wrapWithScrollPane(statePanel), BorderLayout.CENTER)
+                SnykBalloonNotificationHelper.showError(unexpectedErrorMessage, null)
```

Comment:
Is it possible that you wanted to use your new `showErrorBalloonForComponent` function instead of `showError`? It seems that `showErrorBalloonForComponent` doesn't have references 🤔

Reply:
Ah originally I meant to do that while I was figuring out how to create the balloon but what I ended up doing worked better for our case. Good spot!